packages/wp-dashboard/src/components/editorSettings/customFonts/index.js

```diff
@@ -187,47 +187,57 @@ function CustomFontsSettings({
   const handleOnSave = useCallback(async () => {
     if (canSave) {
       const urlWithProtocol = withProtocol(fontUrl);
-      let canFetch = false;
       try {
         await fetch(urlWithProtocol, {
           method: 'HEAD',
         });
-        canFetch = true;
       } catch (err) {
         setInputError(
           __(
             'Please ensure correct CORS settings for allowing font usage on this site.',
```

Comment:
Not for this PR but the error handling here seems off i.e. a 404 will end up with `this could be any error not just a CORS error`

Reply:
How so? 404s are not throwing errors to be caught AFAIK

Reply:
Tested use `https://overpass-30e2.kxcdn.com/overpass-extrabold-italic.tt` vs `https://overpass-30e2.kxcdn.com/overpass-extrabold-italic.ttf`

<img width="1062" alt="Screen Shot 2022-03-22 at 6 10 36 AM" src="https://user-images.githubusercontent.com/62242/159457599-7de873b8-da14-4c47-bf67-459c2cc71f85.png">

Reply:
An never mind --- it is catching for the CORS error